src/common/boards/nrf_PCA10188_0.1.0_54LV10.json

```
@@ -0,0 +1,124 @@
+{
+    "board": {
+        "boardVersion": "PCA10188",
+        "boardRevision": "0.1.0",
+        "boardName": "nRF54LV10 DK (hw rev. 0.1.0)"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ledcontrol",
+            "title": "Power to LEDs",
+            "label": "Disable to have LEDs unpowered (and kept off).",
+            "tooltip": "Enabling this option provides LEDs from LED0 to LED3 with 3.3 V of power.",
+            "enable": {
+                "pin": 45
+            }
+        },
+        {
+            "type": "switch",
+            "id": "swd-control",
+            "title": "Software Debugger (SWD)",
+            "label": "Disable to use an external debugger on the target chip.",
+            "tooltip": "When enabled, the SWD is connected and the IMCU is acting as the debugger. When disabled, the SWD is disconnected and you can use an external debugger connected to the Debug In header.",
+            "enable": {
+                "pin": 6
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ls-en",
+            "title": "Load Switch",
+            "label": "Voltage level shifter",
+            "tooltip": "Enables voltage level shifter between Board Controller and nRF54L MCU when active. When disabled, the voltage level shifter is disconnected.",
```

```suggestion
            "tooltip": "When enabled, connects the voltage level shifter between Board Controller and nRF54L MCU when active. When disabled, the voltage level shifter is disconnected.",
```
"and nRF54L MCU when active" -- when what is active? Board Controller, MCU, or the shifter?

---

Hi, this port activates the UART0 from the nrf54L mcu to the debugger and that's a better naming for this. The HW Designer of this kit is on vacation but that's what I got from one of the designers currently at work.

---

Followed up with @mezaone and "when active" can be dropped. So this is the wording to go for, @KievDevel :
```suggestion
            "tooltip": "When enabled, connects the voltage level shifter between Board Controller and nRF54L MCU. When disabled, the voltage level shifter is disconnected.",
```